src/lib.rs

```diff
@@ -716,6 +692,9 @@ pub trait Log: Sync + Send {
     /// Implementations of `log` should perform all necessary filtering
     /// internally.
     fn log(&self, record: &Record);
+
+    /// Flushes any buffered records.
+    fn flush(&self) {}
```

Comment:
We should maybe not have a default implementation here? `Write::flush` does't have a default as a comparison.